charts/jenkins/VALUES.md

```diff
@@ -0,0 +1,298 @@
+# jenkins
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| additionalAgents | object | `{}` | Configure additional |
```

Comment:
configure additional what.
type object isn't very useful either without an example of the object but maybe thats the best you can do

Reply:
I could add `(see values.yaml for structure)` to everything thats just a plain object (with default commented-out examples)